### PR TITLE
34-DFA

### PR DESCRIPTION
**Pull request by kfahn22**
Adding 34-diffusion-limited-aggregation

### CODE REVIEW

File: content/videos/challenges/34-diffusion-lmited-aggregation/index.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                    "title": "Diffusion-Limited Aggregation on Wikipedia",
+                    "url": "https://en.wikipedia.org/wiki/Diffusion-limited_aggregation",
+                    "description": "Discussion of DFA."
+                },
+                {
+                    "icon": "📚",
+                    "title": "DLA reference",
+                    "url": "http://paulbourke.net/fractals/dla/",
+                    "description": "Article about DLA."
+                }
```
Comment (klinegareth) [inline content/videos/challenges/34-diffusion-lmited-aggregation/index.json:42]: Missing link to [Atom Live Server Package](https://atom.io/packages/atom-live-server)

File: content/videos/challenges/34-diffusion-lmited-aggregation/showcase/contribution1.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "title": "DLA Simulator 2.0",
+  "url": "https://github.com/infernal9999/Diffusion-limited-Aggregation",
+  "author": {
+      "name": " by Diptanu Roy",
```
Comment (klinegareth) [inline content/videos/challenges/34-diffusion-lmited-aggregation/showcase/contribution1.json:5]: Is the author name supposed to be just "Diptanu Roy"?

**Review by klinegareth — CHANGES_REQUESTED** [review]
All good, just needs an additional link and typo fix (?)

File: content/videos/challenges/34-diffusion-lmited-aggregation/showcase/contribution1.json (hunk 2 above)
Comment (kfahn22) [reply content/videos/challenges/34-diffusion-lmited-aggregation/showcase/contribution1.json:5]: I can fix that, sorry I missed it.

File: content/videos/challenges/34-diffusion-lmited-aggregation/index.json (hunk 1 above)
Comment (kfahn22) [reply content/videos/challenges/34-diffusion-lmited-aggregation/index.json:48]: I have added the link to Atom live server.

**Comment by kfahn22** [comment]
This has already been updated.  Please let me know if there is anything you want.

**Review by klinegareth — APPROVED** [review]

**Pull request merged**